### ISSUE TITLE
Semantic Instance Segmentation with a Discriminative Loss Function 

### ISSUE THREAD

**Issue by junxnone** [junxnone/tio#716]
# Reference
- [ ] [paper - 2017 - Semantic Instance Segmentation with a Discriminative Loss Function]()
- [ ] [Github repo](https://github.com/Wizaron/instance-segmentation-pytorch)

**Comment by junxnone** [comment]
#305
